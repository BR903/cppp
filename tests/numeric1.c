#if foo
one
#elif bar
two
#endif
#if foo > bar
three
#elif foo < bar
four
#else
five
#endif
#if foo + bar == foo * bar + 1
six
#endif
