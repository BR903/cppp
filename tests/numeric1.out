/* -Ufoo */
#if bar
two
#endif
#if 0 > bar
three
#elif 0 < bar
four
#else
five
#endif
#if 0 + bar == 0 * bar + 1
six
#endif
/* -Dbar */
#if foo
one
#else
two
#endif
#if foo > 1
three
#elif foo < 1
four
#else
five
#endif
#if foo + 1 == foo * 1 + 1
six
#endif
/* -Dfoo -Dbar */
one
five
six
/* -Ufoo -Ubar */
five
/* -Dfoo=0 -Dbar=0 */
five
/* -Dfoo=1 -Dbar=0 */
one
three
six
/* -Dfoo=0 -Dbar=2 */
two
four
/* -Dfoo=1 -Dbar=2 */
one
four
six
